static/js/app.ts

```diff
@@ -546,10 +546,23 @@ export function viewProgramLink(programId: string) {
   return window.location.origin + '/hedy/' + programId + '/view';
 }
 
+function change_shared (shared: boolean, index: number) {
```

Comment:
What is the meaning of index here? Is it a program id? Of a page element id? In what situations is it -1?

Can we add that as a comment because as it is I don't really understand the meaning.

Reply:
My bad! The index is a front-end element id so we are able to remove / hide this element without have to verify with the server again. Will add some additional line comments.

Reply:
Thanks! I don't look at the front-end code that often so I need a bit of help to stay in the know :)